src/Containers/Microsoft.NET.Build.Containers/ImageReference.cs

```diff
@@ -10,7 +10,7 @@ namespace Microsoft.NET.Build.Containers;
     public override string ToString()
     {
         if (Registry is {} reg) {
-            return $"{reg.BaseUri.GetComponents(UriComponents.HostAndPort, UriFormat.Unescaped)}/{Repository}:{Tag}";
+            return $"{reg.RegistryName}/{Repository}:{Tag}";
```

Comment:
This is a bugfix I noticed along the way - base images were reported with the port specified even if the user hadn't explicitly specified them:

![image](https://user-images.githubusercontent.com/573979/234637164-6c5d104d-4ddc-4f3e-88b8-af0c3ac3c7d3.png)

We tackled this in other locations with the `RegistryName` member, it just needed to be exposed publicly so it could be used here as well.